### PR TITLE
Updates in layout and context

### PR DESCRIPTION
**Pull request by irinaz**
@sherakama 
Hi Shea, 
I added more updates to content type layout, ready for your review
Thanks, Irina

### CODE REVIEW

File: stanford_fellowship.context.inc [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      ),
+      'options' => array(
+        'node_form' => '1',
+      ),
+    ),
+  );
+  $context->reactions = array(
+    'block' => array(
+      'blocks' => array(
+        'ds_extras-fellowship_important_dates' => array(
```
Comment (sherakama) [inline stanford_fellowship.context.inc]: This block does not appear to be in this feature. Not sure where it is or what it is.

File: stanford_fellowship.context.inc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          'module' => 'ds_extras',
+          'delta' => 'fellowship_important_dates',
+          'region' => 'content',
+          'weight' => '-1',
+        ),
+        'ds_extras-eligibility' => array(
```
Comment (sherakama) [inline stanford_fellowship.context.inc:38]: same here. Is this field to block?

File: stanford_fellowship.ds.inc [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+ */
+function stanford_fellowship_ds_layout_settings_info() {
+  $export = array();
+
+  $ds_layout = new stdClass();
+  $ds_layout->api_version = 1;
+  $ds_layout->id = 'node|stanford_fellowship|full';
+  $ds_layout->entity_type = 'node';
+  $ds_layout->bundle = 'stanford_fellowship';
+  $ds_layout->view_mode = 'full';
```
Comment (sherakama) [inline stanford_fellowship.ds.inc]: I have reverted this module several times and I don't this came through properly. The full content view mode is a flat list of fields. Do you have an example of what it should be so I can help figure out what went wrong?

File: stanford_fellowship.features.inc [hunk 4]
```diff
@@ -5,9 +5,28 @@
...
 
 /**
+ * Implements hook_block_class_features_default_class().
+ */
+function stanford_fellowship_block_class_features_default_class() {
+  return array(
+    'ds_extras:fellowship_important_dates' => array(
+      'module' => 'ds_extras',
+      'delta' => 'fellowship_important_dates',
+      'css_classes' => 'pull-right span4  well',
```
Comment (sherakama) [inline stanford_fellowship.features.inc:15]: We don't export block classes in our features as they may change on a per site basis. We end up putting these into the  install profile. I can help with that once this PR is ready to be merged. Leave them in here for now as a reminder.

**Comment by sherakama** [comment]
Thanks for sending this in Irina but it looks like it did not all come through. Can you connect with me to go over this in detail. Thanks.

**Comment by irinaz** [comment]
@sherakama 
Hi Shea, I replaced blocks with groups, can we test that feature builds correctly? 
Thanks, Irina

File: stanford_fellowship.context.inc [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    'node' => array(
+      'values' => array(
+        'stanford_fellowship' => 'stanford_fellowship',
+      ),
+      'options' => array(
+        'node_form' => '1',
+      ),
+    ),
+  );
+  $context->reactions = array(
```
Comment (sherakama) [inline stanford_fellowship.context.inc:29]: @irinaz, None of these blocks exist in this pull request. Is this left over from the first layout structure?

File: stanford_fellowship.ds.inc [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+/**
+ * Implements hook_ds_layout_settings_info().
+ */
+function stanford_fellowship_ds_layout_settings_info() {
+  $export = array();
+
+  $ds_layout = new stdClass();
+  $ds_layout->api_version = 1;
+  $ds_layout->id = 'node|stanford_fellowship|full';
```
Comment (sherakama) [inline stanford_fellowship.ds.inc:15]: This view mode looks to be completely configured but it is not enabled by default. See comment below on strongarm variables.

Comment (sherakama) [inline stanford_fellowship.ds.inc:15]: Here is the line where the 'Full content' view mode is set to disabled by default: https://github.com/SU-SWS/stanford_fellowship/pull/1/files#diff-d6a2bb0339a1b091b37ace1e610c9178L72

File: stanford_fellowship.features.inc (hunk 4 above)
Comment (sherakama) [inline stanford_fellowship.features.inc:15]: Note, this block does not exist.

File: stanford_fellowship.info [hunk 7]
```diff
@@ -13,18 +17,24 @@ dependencies[] = file
 dependencies[] = link
 dependencies[] = list
 dependencies[] = options
+dependencies[] = stanford_affiliate_organization
 dependencies[] = strongarm
 dependencies[] = text
-dependencies[] = stanford_affiliate_organization
 dependencies[] = views
+features[block_class][] = ds_extras:fellowship_important_dates
```
Comment (sherakama) [inline stanford_fellowship.info:25]: Note: remove this on final merge.

**Comment by sherakama** [comment]
Hello @irinaz,

I have pulled down and installed the module. It looks like the 'Full content' display is disabled by default. Please see my comments above.

I was able to create the following:

screen shot 2015-09-23 at 11 07 22 am

Does this look correct?

If so I think it needs a little bit of work. The groups are overflowing one another:
screen shot 2015-09-23 at 12 11 16 pm

and the text flows up extremely close to the right block.
screen shot 2015-09-23 at 12 25 37 pm

I will put this up for review for @meganem. Do we have a mock up for this content type to compare this output to?

Thanks,
Shea

File: stanford_fellowship.context.inc (hunk 5 above)
Comment (irinaz) [inline stanford_fellowship.context.inc:29]: Oh...  Sorry, I totally forgot that I have also added context.  I will remove it and rebuild the feature. Thanks!

File: stanford_fellowship.ds.inc (hunk 6 above)
Comment (irinaz) [inline stanford_fellowship.ds.inc:15]: Full content can be disabled - I am using default layout now - makes it less extra code

File: stanford_fellowship.info (hunk 7 above)
Comment (irinaz) [inline stanford_fellowship.info:25]: got it thanks!  Features will not remove it automatically, right?

**Comment by irinaz** [comment]
Hi Shea, 
We did review general layout with Megan last time.  I plan to meet with Cynthia to review what configuration / classes for groups she will suggest.  I was using standard (pull-right and span4) instead of more custom SWS classes postcard and it's subset. 

Good news for me is that groups and fields are built correctly :)

Thanks, Irina

**Comment by irinaz** [comment]
@sherakama 
I have pushed my updates - hopefully this time they go through.  I ended up adding css file - let me know if that works.

We might have more style comments after Joy and Tori test the feature.
Thanks, Irina

**Comment by sherakama** [comment]
Thanks for this Irina. It is looking pretty good. I made some changes and merged it in. I will rebuild on the vpsa-preview site before the sprint review so we can have a look at it. You should be able to rebuild on your local to get all the changes.

Thanks for all the back and forth and great work!

**Pull request merged**